browser/profiles/profile_util.cc

```diff
@@ -81,31 +34,4 @@ void SetDefaultThirdPartyCookieBlockValue(Profile* profile) {
           content_settings::CookieControlsMode::kBlockThirdParty)));
 }
```

Comment:
we should either move these two left functions out of this file and get rid of it entirely or rename it into `profile_migration_helpers.h/cc` and keep all migration code here explicitly.

Reply:
These are the only two that had tests for them so they couldn't be easily moved like the others, but I'll rename the file for now

Reply:
actually I'm going to leave it for now because I'm not sure migration_helper makes sense here and these methods should definitely move somewhere else in a follow-up

Reply:
https://github.com/brave/brave-browser/issues/41231